src/main/java/com/feedzai/commons/sql/abstraction/dml/dialect/SqlBuilder.java

```diff
@@ -560,6 +560,16 @@ public static View createView(final String name) {
         return new View(name);
     }
 
+    /**
+     * Drops a view.
+     *
+     * @param name The name of the view.
+     * @return The drop view representation.
+     */
+    public static DropView dropView(final String name) {
```

Comment:
The other drop* commands are defined in `AbstractDatabaseEngine`. Did you consider adding this one there also? What would be the pros/cons?